projects/ng-event-plugins/src/plugins/debounce.plugin.ts

```diff
@@ -0,0 +1,64 @@
+import type {EventManager} from '@angular/platform-browser';
+
+import type {EventManagerPlugin} from '../types/event-manager-plugin';
+
+export class DebounceEventPlugin implements EventManagerPlugin {
+    private readonly regExp = /\.debounce-(?<time>\d+)(?<units>ms|s)/;
```

Comment:
Small point, but we can keep it as const REGEXP outside of class, makes it a bit shorter to access (no this.) and the name would be minified. That's how we generally try to approach such things.